events/2022-january-22/mentored_sprint_agenda.md

```diff
@@ -0,0 +1,93 @@
+#  Jupyter Accessibility Workshops: Writing image descriptions for documentation
+
+This event is held one to two weeks following our accessibility learning 
+experience.
+
+## Event details
+
+Date: January 22, 2022
+
+Overview: A hands-on event focused on improvements to the Jupyter code bases and 
+documentation accessibility.
+
+## Takeaways
+
+Attendees will become familiar with the different skills needed to provide a 
+better experience for disabled scientists in the Jupyter ecosystem, especially 
+around documentation and alt text (a common type of image description).
+
+## What you need to participate
+
+* In order to participate in the sprint, attendees will need a GitHub account. 
+No other knowledge of git or GitHub is required.
+    * [Create a free GitHub account here](https://github.com/join). You can find [help for creating an account on WikiHow](https://www.wikihow.com/Create-an-Account-on-GitHub).    
+
+## Schedule
+
+```mermaid
+gantt
+
+    dateFormat YYYY-MM-DD hh:mm
+    axisFormat  %H:%M
+
+    section icebreaker
+    meet and greet :meet, 2021-10-02 08:45, 15min
+    welcome, welcome, welcome :welcome, after meet, 15min
+        
+    section introductions
+    intro to jupyter :jupyter, after welcome, 10min 
+    wcag and accessibility tools :tools, after jupyter, 30min
+            
+    section breakouts
+    breakout topics :topics, after tools, 10min
+    breakout 1 :b1, after topics, 40min
+    breakout 2 :b2, after topics, 40min
+    summarize :regroup, after b1, 10min
+    past a11y issues/prs :past, after regroup, 10min
+    breakout 1 :b21, after past, 45min
+    breakout 2 :b22, after past, 45min
+        
+    section conclusion
+    final comments :end, after b21, 15min
+    
+    section hangout
+    hangout :hang, after end, 45min
+    
+```
+
+* Enter in breakout rooms (like you about to go on an amusement park ride)
+* Welcome, welcome, welcome
+* Intro to Jupyter
+* Intro to accessibility tools (WAVE, JAWS, NVDA)
+    * Demo tools on documentation to show the problem (and solution?)
+* Breakout topics
+    * Writing alt text for [Project Jupyter documentation](https://jupyter.readthedocs.io/en/latest/) 
+    or other Jupyter ecosystem documentation (no git knowledge needed)
+    * Adding automated tests ([pa11y](https://pa11y.org/) and/or 
```

Comment:
I think this is a great topic, but I am concerned about splitting our group. Especially when I don't know who is going to run this breakout room (I don't have the knowledge to do so).

I'm tempted to remove this option so we don't split/spread our few event running-people thin. Thoughts?

If we remove this section, I'd also propose we make this agenda closer to the ones I've been using for similar (smaller) alt text sprints across projects ([here's an example agenda from a prior event](https://hackmd.io/2uOMXV_fT0ycLyo4uH67HQ) for reference). This info has been working well for participants for the past few events.

Reply:
We can keep one single team with the main focus being alt-text. If we find that we do have enough folks to take on the  test tasks we can suggest they form another group

Reply:
I'm going to mark it as optional.

I also think this could be a good topic for the next sprint. If we advertise it with this focus, I think we could get some different people.